images/kube-deploy/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
+
+IMG="gcr.io/k8s-testimages/kube-deploy"
+
+image:
+	docker build --build-arg "IMAGE_ARG=$(IMG):$(VERSION)" -t "$(IMG):$(VERSION)" .
+
+push: image
+	gcloud docker -- push "$(IMG):$(VERSION)"
```

Comment:
see https://github.com/kubernetes/test-infra/pull/6765, let's unify and use `docker push` here

Reply:
done